pkg/raft/tracker/progress.go

```diff
@@ -196,6 +200,14 @@ func (pr *Progress) CanBumpCommit(index uint64) bool {
 	return index > pr.sentCommit && pr.sentCommit < pr.Next-1
 }
 
+// IsFollowerCommitStale returns true if the follower's commit index it less
+// than index.
+// If the follower's commit index is pr.Next-1, it means that sending a larger
```

Comment:
This comment seems to be not true? The `pr.Next` logic sits in `CanBumpCommit`. Did you intend to check it similarly here?